teku/src/main/java/tech/pegasys/teku/cli/subcommand/admin/WeakSubjectivityCommand.java

```diff
@@ -103,11 +105,15 @@ private Database createDatabase(
       final BeaconNodeDataOptions beaconNodeDataOptions,
       final Eth2NetworkOptions eth2NetworkOptions) {
     final Spec spec = eth2NetworkOptions.getNetworkConfiguration().getSpec();
+    final AsyncRunnerFactory asyncRunnerFactory =
+        AsyncRunnerFactory.createDefault(
+            new MetricTrackingExecutorFactory(new NoOpMetricsSystem()));
```

Comment:
hmm, I'm not entirely sure we want to trigger database migration in subcommands  so probably don't want to have to create the actual async runner...  Maybe we do need to pass in the AsyncRunnerFactory so it can create the runner only if a migration is actually required?